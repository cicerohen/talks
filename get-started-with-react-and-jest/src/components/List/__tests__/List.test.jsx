import React from 'react';
import { shallow } from 'enzyme';
import List from '../List';

const minimalProps = {
  children: <li>list item</li>,
};

describe('<List /> Tests', () => {
  
  it('should render without exploding', (done) => {
    const wrapper = shallow(<List {...minimalProps}/>);
    expect(wrapper).toHaveLength(1);
    done();
  });

  it('should render correctly', (done) => {
    const wrapper = shallow(<List {...minimalProps}/>);
    expect(wrapper).toMatchSnapshot();
    done();
  });

});
