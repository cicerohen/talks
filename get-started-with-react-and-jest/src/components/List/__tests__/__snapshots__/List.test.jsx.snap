// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> Tests should render correctly 1`] = `
<ul
  className="root"
>
  <li>
    list item
  </li>
</ul>
`;
